cmd/truss/main.go

```diff
@@ -269,6 +260,11 @@ func parseServiceDefinition(cfg *truss.Config) (deftree.Deftree, *svcdef.Svcdef,
 		return nil, nil, errors.Wrapf(err, "failed to create service definition; did you pass ALL the protobuf files to truss?")
 	}
 
+	// TODO: Remove once golang 1.9 comes out and type aliases solve context vs golang.org/x/net/context
+	if err := rewritePBGoForContext(sd.Service.Name, pbgoPaths); err != nil {
```

Comment:
Is this call the the function itself all that need to be removed once 1.9 launches